Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM nginx:stable-alpine
 
+ARG NGINX_CONF=./nginx.conf
+
 RUN sed -i '1idaemon off;' /etc/nginx/nginx.conf
 
-COPY nginx.conf /etc/nginx/conf.d/default.conf
+COPY ${NGINX_CONF} /etc/nginx/conf.d/default.conf
```

Comment:
I don't know if we have done this in a long time but is the current approach to just manually ship this after it is merged? @Mongey

Reply:
I think so